src/bin/flamegraph.rs

```diff
@@ -31,42 +32,109 @@ struct Opt {
         )
     )]
     colors: Palette,
-    /// set background colors. Gradient choices are yellow (default), blue, green, grey; flat colors use "#rrggbb"
+
+    /// Set background colors. Gradient choices are yellow (default), blue, green, grey; flat colors use "#rrggbb"
     #[structopt(long = "bgcolors")]
     bgcolors: Option<BackgroundColor>,
-    /// colors are keyed by function name hash
+
+    /// Colors are keyed by function name hash
     #[structopt(long = "hash")]
     hash: bool,
-    /// use consistent palette (palette.map)
+
+    /// Use consistent palette (palette.map)
     #[structopt(long = "cp")]
     cp: bool,
 
+    /// Change title text
+    #[structopt(long = "title", default_value = "Flame Chart")]
+    title: String,
+
+    /// Second level title (optional)
+    #[structopt(long = "subtitle")]
+    subtitle: Option<String>,
+
+    /// Width of image
+    #[structopt(long = "width", default_value = "1200")]
+    image_width: usize,
+
+    /// Height of each frame
+    #[structopt(long = "height", default_value = "16")]
+    frame_height: usize,
+
+    /// Omit smaller functions (default 0.1 pixels)
+    #[structopt(long = "minwidth", default_value = "0.1")]
+    min_width: f64,
+
+    /// Font type
+    #[structopt(long = "fonttype", default_value = "Verdana")]
+    font_type: String,
+
+    /// Font size
+    #[structopt(long = "fontsize", default_value = "12")]
+    font_size: usize,
+
+    /// Font width
+    #[structopt(long = "fontwidth", default_value = "0.59")]
+    font_width: f64,
+
+    /// Count type label
+    #[structopt(long = "countname", default_value = "samples")]
+    count_name: String,
+
+    /// Name type label
+    #[structopt(long = "nametype", default_value = "Function:")]
+    name_type: String,
+
+    /// Set embedded notes in SVG
+    #[structopt(long = "notes", default_value = "")]
+    notes: String,
+
```

Comment:
It's a little sad that we end up duplicating all the defaults between here and `Options::default()`. I don't have a good solution for that at the moment though. We'll probably have to introduce some consts somewhere. cc @jasonrhansen. It's fine for this PR though.

Reply:
I think something is possible in structopts, but it will introduce additional complexity which is not good either.

I would not worry because this looks like a strict line between cli options and library options.